uniffi_bindgen/src/bindings/swift/templates/wrapper.swift

```diff
@@ -62,7 +62,9 @@ private let initializationResult: InitializationResult = {
     return InitializationResult.ok
 }()
 
-private func uniffiEnsureInitialized() {
+// Make the ensure init function public so that other modules which have external type references to
+// our types can call it.
+public func {{ ensure_init_fn_name }}() {
```

Comment:
Is this actually necessary? According to https://mozilla.github.io/uniffi-rs/next/types/remote_ext_types.html#foreign-bindings, swift files for multiple crates are expected to be compiled in a single module, so this should be visible anyway (and otherwise, it would have to get imported somehow). Also, flipping it back to `private` does not break your tests.

Reply:
Maybe not, feel free to open a PR that changes this back to private.  The swift docs say "Private access restricts the use of an entity to the enclosing declaration", so is the enclosing declaration the module?  That feels a bit weird to me, but maybe that's how swift works.

Reply:
:thinking: I think you're right, yes. On top-level the surrounding declaration should be the file, so that `private` and `fileprivate` have the same effect. I realized that the test does not actually seem to cover external initialization functions. However, this should still be `internal` instead of `public`.